OptimobileShared/MediaHelper.swift

```diff
@@ -2,17 +2,18 @@
 
 import Foundation
 
-public enum MediaHelper {
-    private static let mediaResizerBaseUrl: String = "https://i.app.delivery"
+enum MediaHelper {
+    /// Use ``Region.US`` as fallback region.
+    static let mediaResizerBaseUrl: String = "https://i-us-east-1.app.delivery"
```

Comment:
huh, why, is it not better to crash?

Reply:
it will crash anyway (hopefully) if you try to request wrong region , just it will be more obscure. If it doesnt crash we will just be silently not sending any events

Reply:
It is a bad way to produce a crash in notification_service_extension. In case of a crash or a timeout limit -- a notification will be shown without an image by the OS.
So, in the worst case, there will be no image attached to the notification.

Ideally, the URL should be placed in the payload.

Reply:
mmm, ok, mb crashing is bad. But i think silently displaying notification without picture (i.e.not how they wanted it) is also quite bad. It's questionable what's better -- display without pic or not display at all.

But at least could add some logging? + still default US seems weird. Is it to make it work sometimes? we could throw and catch

Reply:
For this, agree in the sense having a default that _may_ work sometimes is not perhaps the best.

It could obscure setup problems for US tenants which may affect other things if not correctly configured.

Personally think it is better if the region is missing to show the content without the picture & log a warning about bad init state.

In line with Vlad tho, we may have to ship like this and create a tech debt to discuss.